contribute.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "django-browserid",
+    "description": "django-browserid is a library that integrates BrowserID authentication into Django.",
```

Comment:
Correct! The underlying protocol is called BrowserID. See http://django-browserid.readthedocs.org/en/latest/user/intro.html#persona-dependence for more info.